pom.xml

```diff
@@ -4,7 +4,7 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.eppo.sdk</groupId>
+    <groupId>cloud.eppo</groupId>
     <artifactId>eppo-server-sdk</artifactId>
     <version>1.0.0</version>
```

Comment:
Please also address [this TODO](https://github.com/Eppo-exp/java-server-sdk/blob/main/src/main/java/com/eppo/sdk/EppoClient.java#L189) about the version